gtk4/src/subclass/application.rs

```diff
@@ -48,47 +47,34 @@ impl<T: GtkApplicationImpl> GtkApplicationImplExt for T {
     }
 }
 
-unsafe impl<T: GtkApplicationImpl> IsSubclassable<T> for ApplicationClass {
-    fn override_vfuncs(&mut self) {
-        unsafe extern "C" fn application_window_added<T: GtkApplicationImpl>(
-            ptr: *mut gtk_sys::GtkApplication,
-            wptr: *mut gtk_sys::GtkWindow,
-        ) {
-            let instance = &*(ptr as *mut T::Instance);
-            let imp = instance.get_impl();
-            let wrap: Borrowed<Application> = from_glib_borrow(ptr);
+unsafe impl<T: ApplicationImpl> IsSubclassable<T> for Application {
+    fn override_vfuncs(class: &mut ::glib::Class<Self>) {
+        <glib::Object as IsSubclassable<T>>::override_vfuncs(class);
```

Comment:
```suggestion
        <gio::Application as IsSubclassable<T>>::override_vfuncs(class);
```

:scream:

Reply:
Agh, my bad.

Reply:
Note that there's also some more magic happening there in gtk3, which I guess we'll include as port of going through all commits there and backporting as needed :)